packages/medusa/src/migrations/1657267320181-extended_batch_job.ts

```diff
@@ -0,0 +1,37 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class extendedBatchJob1657267320181 implements MigrationInterface {
+  name = "extendedBatchJob1657267320181"
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // check if batch job table exists and status column exists
+    // if that's the case, the previous migration has already been run
+    const batchJobTableExists = await queryRunner.query(`SELECT exists (
```

Comment:
Suggestion(non blocking): renamed such as batchJobColumnStatusExists since we are looking for the column and we are sure the table exists at that point